src/structures/Message.js

```diff
@@ -333,6 +338,15 @@ class Message extends Base {
     return clone;
   }
 
+  /**
+   * The channel that the message was sent in
+   * @type {TextChannel|DMChannel|NewsChannel|ThreadChannel}
```

Comment:
```suggestion
   * @type {?(TextChannel|DMChannel|NewsChannel|ThreadChannel)}
```

Reply:
we get channels with the initial guild payload or channel creates
unless people severely mess things up (sweep channels during runtime) this can not return null